helm/etcd-backup-operator/templates/crds/etcd-backup.yaml

```diff
@@ -0,0 +1,209 @@
+{{- if not (.Capabilities.APIVersions.Has "backup.giantswarm.io/v1alpha1/ETCDBackup") }}
```

Comment:
Why do we want this? Don't we want to upgrade the CRD in case we modify it?

Reply:
yes, but how about migration?
Currently the CRD is installed with `opsctl ensure` and installation would fail.
I could remove the check, but that would mean deleting the crd from all MCs by hand.

WDYT?

Reply:
Yeah thinking about it there is no way around it. Thanks for pointing that out. Updated